### ISSUE TITLE
totem buffs dont get extra stats from kings

### ISSUE THREAD

**Issue by guruvex** [slowtorta/turtlewow-bug-tracker#2907]
**Please, provide a clear description what the bug is:**
the grace of air totem buff does not benefit from the kings buff.
 
**Steps to reproduce the behavior with as much detail as possible:**

1.have grace of air totem buff. have pally put kings on you.
2. the math shows the totem is still giving 77 agi.
3. 

**IDs of affected creatures, items, quests or spells with a link to the relevant page:**

NPC: https://database.turtle-wow.org/?npc=
Item: https://database.turtle-wow.org/?item=
Quest: https://database.turtle-wow.org/?quest=
Spell: https://database.turtle-wow.org/?spell=
Object: https://database.turtle-wow.org/?object=

**Expected behavior. Describe how it should work:**
talking to moon the developer he says it should get more than the base 77 agi.
the ZG heart buff gives extra agi for totem buffs and im told kings is the same thing. just 5% less.
so kings should be giving grace of air totem buff more agi.

**Comment by Zeyla-TW** [comment]
Just tested, with a base of 150 agility and kings i went up to 165 agility, which is a 15 difference.
With a 77 agility totem in my party, my agility went up to 227, and kings pushed it to 249.
249-227=22 difference in agility, instead of the previous 15 difference it would have been if kings were not affecting grace of air totem agility. 22-15 = 7 which is the rounded down value of a 10% increase on 77 grace of air.

To summarize, Blessing of Kings is currently multiplying the agility from Grace of Air.

**Comment by Haaxor1689** [comment]
works as intended
